README.md

```diff
@@ -233,9 +233,10 @@ possible security implications.
 
 ### Memory Features
 
-| Feature name   | Description                                                                         |
-| :------------: | :---------------------------------------------------------------------------------: |
-| numa           | Multiple memory nodes i.e. NUMA architecture detected
+| Feature name          | Description                                              |
+| -------------------   | -------------------------------------------------------- |
+| numa                  | Multiple memory nodes i.e. NUMA architecture detected
+| transparent_hugepage  | Check Transparent hugepage is enabled or not
```

Comment:
Please list the attributes (enabled, defrag) in the table, too. Similar table as "Network Features" has, for example. "numa" would have no attribute, of course (i.e. the cell will be empty)

Reply:
Done.